cmd/dep/main.go

```diff
@@ -23,173 +24,186 @@ var (
 	treeOutput       bool
 	outputFile       string
 	depLabelSelector string
-
-	rootCmd = &cobra.Command{
-		Use:   "analyze",
-		Short: "Tool for working with analyzer-lsp",
-		Run:   func(c *cobra.Command, args []string) {},
-	}
 )
 
 func init() {
-	rootCmd.Flags().StringVar(&providerSettings, "provider-settings", "provider_settings.json", "path to the provider settings")
-	rootCmd.Flags().BoolVar(&treeOutput, "tree", false, "output dependencies as a tree")
-	rootCmd.Flags().StringVar(&outputFile, "output-file", "output.yaml", "path to output file")
-	rootCmd.Flags().StringVar(&depLabelSelector, "dep-label-selector", "", "an expression to select dependencies based on labels provided by the provider")
 }
 
-func main() {
-	if err := rootCmd.Execute(); err != nil {
-		println(err.Error())
-	}
-
-	logrusLog := logrus.New()
-	logrusLog.SetOutput(os.Stdout)
-	logrusLog.SetFormatter(&logrus.TextFormatter{})
-	log := logrusr.New(logrusLog)
-
-	logrusErrLog := logrus.New()
-	logrusErrLog.SetOutput(os.Stderr)
-	errLog := logrusr.New(logrusErrLog)
-
-	err := validateFlags()
-	if err != nil {
-		errLog.Error(err, "failed to validate input flags")
-		os.Exit(1)
-	}
+func DependencyCmd() *cobra.Command {
+
+	var errLog logr.Logger
+	rootCmd := &cobra.Command{
+		Use:   "konveyor-analyzer-dep",
+		Short: "Tool for working with dependency for konveyor-analyzer",
```

Comment:
I feel like this description could be worded differently, but nothing major. Otherwise LGTM

Reply:
updated